Automattic/wp-calypso
# Add Missing Credentials Warning to Backups Page

#### Changes proposed in this Pull Request

1. Add warning to backups page when credentials are missing.

<img width="350" src="https://user-images.githubusercontent.com/2810519/77486377-d9159080-6dec-11ea-83d8-c3d068e675c9.png" >

#### Testing instructions

1. Navigate to `/backups`, and chose any site with backups
2. Use redux tools to dispatch the following action:
```
{
  type: 'REWIND_STATE_UPDATE',
  siteId: 173211884,
  data: {
    state: 'awaiting_credentials',
  }
}
```
where the `siteId` is the id of your site.
3. Verity the component appears on the page and matches the above screenshot
